app/qml/form/FeatureForm.qml

```diff
@@ -12,8 +12,6 @@ import QtQuick.Controls 2.0
 import QtQml.Models 2.2
 import QtQml 2.2
 
-// We use calendar in datetime widget that is not yet implemented in Controls 2.2
-import QtQuick.Controls 1.4 as Controls1
```

Comment:
so where is the new calendar? do not see qml here

Reply:
This include was not used anywhere.

Calendar is included from `controls1` in `inputdatetime.qml`, I tested it on multiple devices and it seems that using controls1 does not break the scaling. Calendar, however, have ux issues with hidpi scaling, so I created a ticket to implement a new one once this PR is merged: https://github.com/lutraconsulting/input/issues/1774